python/django_bolt/_bridge.py

```
@@ -0,0 +1,95 @@
+returns ``None`` inside the FIRST segment (before the first real suspension).
+After the first suspension a real Task exists. Set
+``DJANGO_BOLT_EAGER_DISPATCH=0`` to restore the previous Task-per-request
+bridge if a library depends on that.
+"""
+
+from __future__ import annotations
+
+import asyncio
+
+
+async def _drive_started(coro, pending):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_drive_started`

(ANN202)